packages/react/src/useFetch.ts

```diff
@@ -122,22 +145,55 @@ export function useFetch<T = string>(path: string, options?: FetchHookOptions):
           throw new Error(response.statusText);
         }
 
+        let dispatchData = true;
+
         if (mergedOptions.json) {
           data = await response.json();
+        } else if (typeof mergedOptions.stream === "string") {
+          dispatchData = false;
+          const decodedStream = response.body!.pipeThrough(
+            new TextDecoderStream(mergedOptions.stream === "string" ? "utf8" : mergedOptions.stream)
```

Comment:
I think to be a little more beginner friendly so that they don't have to think about encodings and just say: "hey give me this AI streaming response as a string I can display please"